### ISSUE TITLE
Main image comes from wrong occurrence records

### ISSUE THREAD

**Issue by charvolant** [AtlasOfLivingAustralia/bie-index#200]
Both https://bie.ala.org.au/species/NZOR-4-87352 and https://bie.ala.org.au/species/http://id.biodiversity.org.au/node/apni/2908665 have the same main image. This will be because of a scientific name search during loading.

We do the name search when the BIE is first loaded with a new names list, since occurrences have not yet been reprocessed to match the new taxonomy. If we strictly go by taxonID then we'll need to rebuild the BIE after reprocessing.

**Comment by charvolant** [comment]
See, also

Hello
I noticed just now that my photo illustrating Acrotriche divaricata as a result of a search on that name has not yet been removed, though it has been removed from the sighting. 
I drew the attention of ALA a month or so ago to the fact that at the time the photo was taken, Acrotriche leucocarpa had not been seggregated from Acrotriche divaricata. The photo is actually of Acrotriche leucocarpa. I have made a comment to that effect in the comments field of the Acrotriche divaricata sighting.
Regards
Betty Wood

**Comment by charvolant** [comment]
Hello again
I have checked my paper copy of Flora of NSW, and now understand what happened. At the time the photograph was taken, Acrotriche leucocarpa was regarded as possibly in NSW, all sightings being referred to Acrotriche divaricata. I have added a notation to the sighting uploaded to ALA as Astrotriche divaricata that is in our (Don and Betty Wood’s) set of sightings (NSW South Coast & ACT plants) about the taxonomic issue - that it is now Acrotriche leucocarpa. This is exactly the same sighting recently uploaded to ALA as Acrotriche leucocarpa from Atlas of Life in the Coastal Wilderness with the notation that it was supplied by Betty and Don Wood.
What a mess.
Regards
Betty Wood

The Acrotriche leucocarpa link uploaded from the Atlas of Life in the Coastal Wilderness:
https://biocache.ala.org.au/occurrences/5cb85e74-9529-416b-a906-0b6fb81edff2

The Acrotriche divaricata links uploaded from NSW South Coast & ACT Plants
https://biocache.ala.org.au/occurrences/84a26228-06cb-478a-a44b-40187646af41

https://biocache.ala.org.au/occurrences/bf54056c-736c-46b6-885a-3e15722412ec

https://biocache.ala.org.au/occurrences/64f8314e-e845-4ce1-9ddd-088d8ec34837
